crates/ra_hir_ty/src/tests.rs

```diff
@@ -349,3 +349,63 @@ fn no_such_field_with_feature_flag_diagnostics() {
 
     assert_snapshot!(diagnostics, @r###""###);
 }
+
+#[test]
```

Comment:
I think we can now remove the test before this?

Reply:
I think no, it's not the same case. Here the attributes are caught on the function. Not the others